### ISSUE TITLE
React Container throws errors if tooltips, widgets, and default_wrappers are not defined in config

### ISSUE THREAD

**Issue by bramski** [michelson/dante2#93]
I find I have to do the following:

```
const danteConfig = {debug: true, read_only: true, tooltips: [], widgets: [],default_wrappers: []};
<DanteEditor config={danteConfig} content={widgetData.editor_content} />
```
Otherwise the editor crashes.  Shouldn't these just be defaulted to empty lists?

**Comment by michelson** [comment]
yes indeed, that's because the editor iterates over widgets and is expecting tooltips & widgets to be an iterable object, I guess the error is not very semantic

**Comment by bramski** [comment]
The fact that it's a ```config``` variable is difficult in my opinion.  If instead of using ```config``` the options were just provided at the top level as such...
```<DanteEditor read-only debug tooltips={myTooltips}>```
Then they could be

1. Effectively defaulted to empty lists.
2. The writing of the component would be much cleaner.

**Comment by michelson** [comment]
Maybe we can just rescue the empty prop and default it to an empty array, that way it work seamlessly

**Comment by Bouncey** [comment]
Adding to this (thanks @bramski for the help in getting something rendered), there should be sensible defaults for all the things. At the minute my console is a sea of TypeErrors

```
TypeError: this.character_convert_mapping is undefined
TypeError: tooltip is undefined
TypeError: oldNode.parentNode is null
```

Honestly, I thank you for your time in creating this project and having it open source. But better docs (#108) and/or sensible defaults would be great to lower the barrier of entry.

**Comment by michelson** [comment]
fixed in 0.5.0-rc1 install as Dante2@next
